src/faq.html

```diff
@@ -0,0 +1,38 @@
+---
+layout: default.html
+pageTitle: FAQ
+tags: nav
+order: 3
+---
+
+
+
+
+<ul class="u-list-reset faq-set">
+  
+  <h1 class="faq-header">Frequently Asked Questions</h1>
+  {% include faq.html %}
+</ul>
+
+
+
+<h3 class="form-header">If you would like to ask a question fill out this form</h3>
+
+<div class="container">
+  <form action="/action_page.php">
+    <label for="fname">First Name</label>
+    <input type="text" id="fname" name="firstname" placeholder="Your name..">
+
+    <label for="lname">Your Email to get back to you</label>
+    <input type="text" id="lname" name="lastname" placeholder="Email">
```

Comment:
There's a specific `type` for email addresses:
```suggestion
    <input type="email" id="lname" name="lastname" placeholder="Email">
```

Learn more about valid `type` attributes for the `input` element here: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input#input_types